doc/install.rst

```diff
@@ -41,19 +41,64 @@ This will install the latest stable release along with all the dependencies.
    you haven't already done so
 -  Get an API token from the IBM Q experience website under “My
    Account” > “Personal Access Token”
--  The API token needs to be placed in a file called ``Qconfig.py``. For
-   convenience, we provide a default version of this file that you
-   can use as a reference: `Qconfig.py.default`_. After downloading that
-   file, copy it into the folder where you will be invoking the SDK (on
-   Windows, replace ``cp`` with ``copy``):
+
+3.1 Store API credentials locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For most users, storing your API credentials is most easily done locally.
+Your information is stored locally in a configuration file called `qiskitrc`.
+To store your information, simply run:
+
+.. code:: python
+
+    from qiskit import store_credentials
+
+    store_credentials(`MY_API_TOKEN`)
+
+where `MY_API_TOKEN` should be replaced with your token.
+
+If you are on the IBM Q network, you must also pass `url`,
+`hub`, `group`, and `project` arguments to `store_credentials`.
```

Comment:
1) What to do if multiple IBMQProviders are created. 2) How best to search through multiple registered providers [LocalProvider, IBMQProvider1, IBMQProvider2, ExternalProvider1,...] so that double registration can not happen, and we know which providers are loaded so that one or more may be unregistered.  Unregister was requested elsewhere.